recipes/avahi/system/conanfile.py

```diff
@@ -0,0 +1,44 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class SysConfigAvahiSystemConan(ConanFile):
+    name = "avahi"
+    version = "system"
+    description = "Conan package for Avahi - Service Discovery for Linux using mDNS/DNS-SD - compatible with Bonjour"
+    topics = ("Avahi", "Bonjour", "DNS-SD", "mDNS")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/lathiat/avahi"
+    license = "LGPL-2.1-only"
+    settings = "os"
+
+    def validate(self):
+        if self.settings.os != "Linux" or tools.cross_building(self):
+            raise ConanInvalidConfiguration("Only Linux is supported for this package.")
+
+    def system_requirements(self):
+        packages = []
+        if self.settings.os == "Linux" and tools.os_info.is_linux:
+            # for building against Avahi Apple Bonjour compatibility library, libdns_sd and dns_sd.h are required
+            # the Name Service Switch integration and command-line utilities are useful
+            if tools.os_info.with_apt:
+                # Debian/Ubuntu
+                packages = ["libavahi-compat-libdnssd-dev", "libnss-mdns", "avahi-utils"]
+            elif tools.os_info.with_yum:
+                # RHEL/CentOS/Fedora
+                packages = ["avahi-compat-libdns_sd-devel", "nss-mdns", "avahi-tools"]
```

Comment:
```suggestion
                packages = ["avahi-compat-libdns_sd-devel", "nss-mdns"]
```
[centos8](https://github.com/bincrafters/system-packages-checks/runs/3380619317?check_suite_focus=true#step:6:343) fails with `Error: Unable to find a match: avahi-tools`. This does not seem to be available : https://forums.centos.org/viewtopic.php?t=71858. Do you think we can simply remove it ?

Reply:
Frustrating, thanks for finding the back story! I have simply removed tools/utils for all distros for now for consistency. The important thing is being able to compile against _dns_sd.h_ and link against _libdns_sd_.